Content.Server/Atmos/EntitySystems/GasTankSystem.cs

```diff
@@ -51,6 +55,18 @@ public override void Initialize()
             SubscribeLocalEvent<GasTankComponent, GasAnalyzerScanEvent>(OnAnalyzed);
             SubscribeLocalEvent<GasTankComponent, PriceCalculationEvent>(OnGasTankPrice);
             SubscribeLocalEvent<GasTankComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAlternativeVerb);
+            _cvar.OnValueChanged(CCVars.AtmosTankFragment, UpdateFragment, true);
+        }
+
```

Comment:
There's a helper method for this, subs.something.

Reply:
Thy will is done.